scripts/install-worker.sh

```diff
@@ -69,6 +71,13 @@ sudo yum install -y \
     yum-plugin-versionlock \
     yum-utils
 
+# Install AWS CLI v2
+AWSCLIv2_DIR=$(mktemp -d)
+curl -L "https://awscli.amazonaws.com/awscli-exe-linux-${MACHINE}.zip" -o "${AWSCLIv2_DIR}/awscliv2.zip"
+unzip "${AWSCLIv2_DIR}/awscliv2.zip" -d ${AWSCLIv2_DIR}
+sudo "${AWSCLIv2_DIR}/aws/install"
+sudo mv /usr/local/bin/aws /usr/local/bin/awsv2
```

Comment:
Unfortunately, we can't do this; because the image has to be buildable in airgapped environments. (We're working on getting the CLI updated in the AL yum repos)